datajunction-clients/python/datajunction/client.py

```diff
@@ -50,58 +51,59 @@ def list_metrics(self, namespace: Optional[str] = None) -> List[str]:
             )
         return self._session.get("/metrics/").json()
 
+    def list_cubes(self, namespace: Optional[str] = None) -> List[str]:
```

Comment:
Unrelated to this PR, but I wonder if something like `find_cube_by_metrics_and_dimensions` would be useful...

Reply:
I am not sure on this. Feels like unless you know of a cube with some content of interest, trying to find cube by providing metrics and dimensions could be a little like shooting in the dark... unless we could provide a list of cubes ordered by closest coverage. Still, I think this is something we may need to see needed first.

Reply:
Instead of by metrics and dimensions, I think by a single metric _or_ by a single dimension would definitely be useful. In other words, "show me what cubes use this single metric" would be great for exploring.